docs/scenario-yaml-format.md

```diff
@@ -1,4 +1,4 @@
-Hints YAML file Format
+Scenario YAML file Format
```

Comment:
note: to change to tasks.yaml

Reply:
Thanks.  I hadn't noticed we'd used that name for the kubernetes deployments.  I'll rename again to `tasks.yaml` and update the docs